src/DssSpell.sol

```diff
@@ -15,216 +16,196 @@
-
-        CollateralOpts memory MATIC_A = CollateralOpts({
-            ilk:                   "MATIC-A",
-            gem:                   MATIC,
-            join:                  MCD_JOIN_MATIC_A,
-            clip:                  MCD_CLIP_MATIC_A,
-            calc:                  MCD_CLIP_CALC_MATIC_A,
-            pip:                   PIP_MATIC,
-            isLiquidatable:        true,
-            isOSM:                 true,
-            whitelistOSM:          true,
-            ilkDebtCeiling:        3 * MILLION,
-            minVaultAmount:        10 * THOUSAND,
-            maxLiquidationAmount:  3 * MILLION,
-            liquidationPenalty:    1300,
-            ilkStabilityFee:       THREE_PCT_RATE,
-            startingPriceFactor:   13000,
-            breakerTolerance:      5000, // Allows for a 50% hourly price drop before disabling liquidations
-            auctionDuration:       140 minutes,
-            permittedDrop:         4000,
-            liquidationRatio:      17500,
-            kprFlatReward:         300,
-            kprPctReward:          10 // 0.1%
-        });
-
-        DssExecLib.addNewCollateral(MATIC_A);
-        DssExecLib.setIlkAutoLineParameters("MATIC-A", 10 * MILLION, 3 * MILLION, 8 hours);
-
-        DssExecLib.setChangelogAddress("MATIC", MATIC);
-        DssExecLib.setChangelogAddress("MCD_JOIN_MATIC_A", MCD_JOIN_MATIC_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_MATIC_A", MCD_CLIP_MATIC_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_MATIC_A", MCD_CLIP_CALC_MATIC_A);
-        DssExecLib.setChangelogAddress("PIP_MATIC", PIP_MATIC);
-
-
-        // Liquidation Ratio
-        // https://vote.makerdao.com/polling/QmZQdJpG?network=mainnet#poll-detail
-        DssExecLib.setIlkLiquidationRatio("ETH-A", 14500);
-        DssExecLib.setIlkLiquidationRatio("WBTC-A", 14500);
-        DssExecLib.setIlkLiquidationRatio("ETH-C", 17000);
-        DssExecLib.setIlkLiquidationRatio("LINK-A", 16500);
-        DssExecLib.setIlkLiquidationRatio("UNIV2DAIETH-A", 12000);
-        DssExecLib.setIlkLiquidationRatio("YFI-A", 16500);
-        DssExecLib.setIlkLiquidationRatio("UNIV2WBTCETH-A", 14500);
-        DssExecLib.setIlkLiquidationRatio("UNIV2UNIETH-A", 16000);
-        DssExecLib.setIlkLiquidationRatio("UNIV2USDCETH-A", 12000);
-        DssExecLib.setIlkLiquidationRatio("RENBTC-A", 16500);
-        DssExecLib.setIlkLiquidationRatio("UNI-A", 16500);
-        DssExecLib.setIlkLiquidationRatio("AAVE-A", 16500);
-        DssExecLib.setIlkLiquidationRatio("UNIV2WBTCDAI-A", 12000);
-        DssExecLib.setIlkLiquidationRatio("BAL-A", 16500);
-        DssExecLib.setIlkLiquidationRatio("COMP-A", 16500);
-
-
-        // Housekeeping
-        DssExecLib.setChangelogAddress("CALC_FAB", CALC_FAB);
+        // Increase PAX-PSM-A DC from 50 million DAI to 500 million DAI
+        DssExecLib.increaseIlkDebtCeiling("PSM-PAX-A", 450 * MILLION, true);
```

Comment:
```suggestion
        DssExecLib.increaseIlkDebtCeiling("PSM-PAX-A", 500 * MILLION, true);
```

Reply:
Hmm no, this one is fine. It will increase the debt ceiling by 450M placing the new one to 500M.